src/CoreWCF.Primitives/src/CoreWCF/Configuration/ServiceConfiguration.cs

```diff
@@ -27,7 +29,18 @@ public List<IServiceDispatcher> GetDispatchers()
         {
             if (_dispatchers == null)
             {
-                _dispatchers = DispatcherBuilder.BuildDispatcher<TService>(this, _services);
+                // The services can't be Open'd until after ASP.NET Core is running and listening for requests. The Open call
+                // is started and allowed to run in the background. If a request comes in before the dispatchers have finished
```

Comment:
did you run into this issue in your testing ?

Reply:
Yes. It's a hard race condition to hit, but when setting the right break points while debugging, you can hit it reliably.